lib/codegen/fromcto/java/javavisitor.js

```diff
@@ -374,6 +379,16 @@ class JavaVisitor {
     capitalizeFirstLetter(s) {
         return s.charAt(0).toUpperCase() + s.slice(1);
     }
+
+    /**
+     * Check if field is a Map Declaration
+     * @param {Field} field - the field being visited
+     * @return {boolean} true if field is a Map Declaration
+     * @private
+     */
+    isMap(field) {
+        return (Object.keys(field).length > 0 && ModelUtil.isMap?.(field)) ? true : false;
```

Comment:
Tertiary operator here is redundant, you can just return result of the comparison